libs/hdf-converters/src/msft-config-mapper.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  SecureScore,
+  ControlScore,
+  SecureScoreControlProfile
+} from '@microsoft/microsoft-graph-types';
+import {ExecJSON} from 'inspecjs';
+
+import {version as HeimdallToolsVersion} from '../package.json';
+
+import {BaseConverter, ILookupPath, MappedTransform} from './base-converter';
+
+export class MsftConfigMapper extends BaseConverter {
```

Comment:
strange that the mapper is called 'msgftconfigmapper' whereas the other things like the fingerprinting enum and the profile.name refer to it as 'microsoft secure score'.  probably better to consolidate on the one.

Reply:
rename completed